### PR TITLE
fix(docs): changed the name from core v2 to Core 2

### PR DESCRIPTION
**Pull request by Rajakavitha1**
changes the name from seldon core v2 to Seldon Core 2

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by paulb-seldon — APPROVED** [review]
LGTM

File: operator/licenses/license.txt [hunk 1]
```diff
@@ -531,27 +531,27 @@ Azure/go-autorest  Apache License 2.0  https://github.com/Azure/go-autorest/blob
 --------------------------------------------------------------------------------
 AzureAD/microsoft-authentication-library-for-go  MIT License  https://github.com/AzureAD/microsoft-authentication-library-for-go/blob/main/LICENSE
 --------------------------------------------------------------------------------
-    MIT License
```
Comment (sakoush) [inline operator/licenses/license.txt:534]: this is an autogenerated file, revert change.

File: scheduler/licenses/license.txt [hunk 2]
```diff
@@ -531,27 +531,27 @@ Azure/go-autorest  Apache License 2.0  https://github.com/Azure/go-autorest/blob
 --------------------------------------------------------------------------------
 AzureAD/microsoft-authentication-library-for-go  MIT License  https://github.com/AzureAD/microsoft-authentication-library-for-go/blob/main/LICENSE
 --------------------------------------------------------------------------------
-    MIT License
```
Comment (sakoush) [inline scheduler/licenses/license.txt:534]: this is an autogenerated file, please revert.

**Review by sakoush — APPROVED** [review]
LGTM. left minor comments.

File: operator/licenses/license.txt (hunk 1 above)
Comment (Rajakavitha1) [reply operator/licenses/license.txt:534]: Hi @sakoush  I made no changes to those files content. Unable to revert. Please help.

Comment (Rajakavitha1) [reply operator/licenses/license.txt]: just reversed the changes. Please recheck

File: scheduler/licenses/license.txt (hunk 2 above)
Comment (Rajakavitha1) [reply scheduler/licenses/license.txt]: just reversed the changes. Please recheck

**Review by sakoush — APPROVED** [review]
lgtm

**Pull request merged**